### DISCUSSION TITLE
Captured traffic to (skeleton) DBC?

### DISCUSSION THREAD

**Discussion by joehodgy** [collin80/SavvyCAN#576] [Q&A]
Maybe I'm missing it - is there a way to create a bare bones DBC using already captured traffic?

Just a list of all IDs on each bus plugged straight into a DBC would be a great time-saver!

Thanks

Joe

**Comment by collin80** [comment]
There isn't currently a way to do that. I can see how that would save time but it doesn't do that yet. It's a decent idea so I'll think about implementing it.

**Reply by joehodgy** [reply]
Thanks Collin :)
